src/main/MQ2ImGuiConsole.cpp

```diff
@@ -245,6 +251,18 @@ static std::pair<std::string_view, ImU32> ParseColorTags(std::string_view line,
 	return { { pos, (size_t)(end - pos) }, color };
 }
 
+struct FontSizeInfo
+{
+	std::string label;
+	int value;
+};
+
+std::vector<FontSizeInfo> s_fontSizeInfo = {
+	{"10", 10}, {"12", 12}, {"13", 13}, {"14", 14}, {"16", 16},
+	{"18", 18}, {"20", 20}, {"22", 22}, {"24", 24}, {"26", 26},
+	{"28", 28}, {"30", 30}
+};
```

Comment:
why is this a vector of tuples and not a map?  Also, I'm assuming this is in code because the allowable values are baked in, but it seems like you're adding another maintenance point for adding a font size.

Reply:
One of these values is derived from the other. We don't need both. A vector of ints would work just as well.